.github/workflows/package-json-alpha-check.yml

```diff
@@ -0,0 +1,39 @@
+name: Check for Alpha Techsee Packages
```

Comment:
rename: 'Gate-check against merging alpha versions of TechSee packages as dependencies'